tokio-macros/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 1.4.0 (September 29th, 2021)
+### Changed
+
+- macros: run current_thread inside LocalSet ([#4027])
+- macros: explicitly relaxed clippy lint for `.expect()` in runtime entry macro ([#4030])
+
+### Fixed
+
+- macros: fix invalid error messages in functions wrapped with `#[main]` or `#[test]` ([#4067])
```

Comment:
You need to actually include the lines that define the links, e.g.:
```
[#4067]: https://github.com/tokio-rs/tokio/pull/4067
```
This is also why the PR description is wonky.

Reply:
Thanks for catching this, I've updated the PR.